app/uk/gov/hmrc/cbcr/controllers/test/TestSubscriptionDataController.scala

```diff
@@ -85,4 +88,27 @@ class TestSubscriptionDataController @Inject()(subRepo: SubscriptionDataReposito
     reportingEntityDataRepo.delete(DocRefId(docRefId)).map(wr => if(wr.n == 0) NotFound else Ok)
   }
 
+  def updateReportingEntityCreationDate(docRefId:String, creationDate: String) = Action.async {
+    implicit request => {
+      val dri = DocRefId(docRefId)
+      val cd = LocalDate.parse(creationDate)
+
+      reportingEntityDataRepo.updateCreationDate(dri, cd).map {
+        case true => Ok
+        case false => NotModified
```

Comment:
update.ok doesn't say if anything has been modified, it just says nothing broke. Check out update.n which indicates the number of documents modified.

Reply:
now using update.nModified